x-pack/elastic-agent/magefile.go

```diff
@@ -326,6 +326,9 @@ func Package() {
 
 func requiredPackagesPresent(basePath, beat, version string, requiredPackages []string) bool {
 	for _, pkg := range requiredPackages {
+		if _, ok := os.LookupEnv(snapshotEnv); ok {
```

Comment:
@simitt Depending on how soon this PR will get merged, should we get this into a separate PR + some parts of the refactoring / cleanup?

Reply:
yeah I can pull it out if you prefer, but I'd expect to merge this PR in soon.